tests/testthat/test-qenv_get_code.R

```diff
@@ -58,7 +58,7 @@ testthat::test_that("get_code returns code with comments and empty spaces", {
     # closing comment
     "
   q <- eval_code(qenv(), code)
-  testthat::expect_equal(get_code(q), code)
+  testthat::expect_equal(get_code(q), gsub(";", "\n", code))
```

Comment:
When you run

```r
code <- "
    # header comment after white space
    a <- 1L; b <- 2 #inline comment
    c <- 3
    # closing comment
    "
q <- eval_code(qenv(), code)
cat(get_code(q))

```

You can see that indentation for `b` is different than the rest of the code

```r

    # header comment after white space
    a <- 1L
 b <- 2 #inline comment
    c <- 3
    # closing comment
    
```

Should we substitute `;` with `\n` but prefixed with the number of whitespaces that existed at the beginning of the line in which `;` appeared? So that the call looks like this

```r

    # header comment after white space
    a <- 1L
    b <- 2 #inline comment
    c <- 3
    # closing comment
    
```

Reply:
How can you determine the indent here? Previous of `b` is `\n    # header comment after white space\n    a <- 1L`. Here we would need to:
```
call_previous_lines <- split(call[i-1], "\n")
previous_line <- call_previous_lines[length(call_previous_lines)]
previous_line_indent <- gsub("^([\t ]*).*", "\\1", previous_line)
# here we need also to substract one space as 'b' call starts with a single space
paste0(previous_line_indent, call[i])
```

But it doesn't solve all the problems. Because when one wants to pull only the code of `b`, then it would have a indent given by previous (possibly unrelated line).

Reply:
Thanks for looking into that and for the explanation.
Not ideal but I think we can live with those limitations : )

So this is ready to be merged